ArchiSteamFarm/ASF.cs

```diff
@@ -131,27 +130,7 @@ internal static class ASF {
 					return null;
 				}
 
-				string releaseURL = SharedInfo.GithubReleaseURL + (Program.GlobalConfig.UpdateChannel == GlobalConfig.EUpdateChannel.Stable ? "/latest" : "?per_page=1");
-
-				GitHub.ReleaseResponse releaseResponse;
-
-				if (Program.GlobalConfig.UpdateChannel == GlobalConfig.EUpdateChannel.Stable) {
-					WebBrowser.ObjectResponse<GitHub.ReleaseResponse> objectResponse = await Program.WebBrowser.UrlGetToJsonObject<GitHub.ReleaseResponse>(releaseURL).ConfigureAwait(false);
-					if (objectResponse?.Content == null) {
-						ArchiLogger.LogGenericWarning(Strings.ErrorUpdateCheckFailed);
-						return null;
-					}
-
-					releaseResponse = objectResponse.Content;
-				} else {
-					WebBrowser.ObjectResponse<List<GitHub.ReleaseResponse>> objectResponse = await Program.WebBrowser.UrlGetToJsonObject<List<GitHub.ReleaseResponse>>(releaseURL).ConfigureAwait(false);
-					if ((objectResponse?.Content == null) || (objectResponse.Content.Count == 0)) {
-						ArchiLogger.LogGenericWarning(Strings.ErrorUpdateCheckFailed);
-						return null;
-					}
-
-					releaseResponse = objectResponse.Content[0];
-				}
+				GitHub.ReleaseResponse releaseResponse = await GitHub.GetLatestRelease(Program.GlobalConfig.UpdateChannel == GlobalConfig.EUpdateChannel.Stable).ConfigureAwait(false);
```

Comment:
Be careful when you edit existing code and introduce potential regressions. This will cause ASF **to crash** on remote failure because `releaseResponse` can be `null`. Add appropriate `if` and include appropriate `Strings.ErrorUpdateCheckFailed` that you happily removed.